### DISCUSSION TITLE
stop this libplacebo patch from merging, it makes hdr-tonemapping worse

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#15375] [Q&A] [answered]
Hi, can someone stop [this patch](https://code.videolan.org/videolan/libplacebo/-/merge_requests/683)? I thought that it would help but after testing it makes things worse and I can't log in on videolan.
Here you can read more about it https://github.com/haasn/libplacebo/issues/304#issuecomment-2495111635

Thank you :)

**Comment by kasper93** [comment, marked as answer]
cc @haasn

**Reply by geextahslex** [reply]
it worked, thank you :)
